### PR TITLE
Update to GNOME 3.38 and ffmpeg 20.08

### PR DESCRIPTION
**Pull request by Eonfge**
Also removed the gst-python, but we'll have to discuss this because I'm not certain if we still need it.

### CODE REVIEW

File: org.gnome.Lollypop.json [hunk 1]
```diff
@@ -51,19 +51,6 @@
         "sha256": "5500415b865e8b62775d4742cbb9f37146a50caecfc0e7a6fc0160d3c560fbca"
       }]
     },
-    {
-      "name": "gst-python",
```
Comment (bilelmoussaoui) [inline org.gnome.Lollypop.json:55]: Any reason for removing this one? :)

Comment (Eonfge) [reply org.gnome.Lollypop.json:55]: It didn't compile so I wanted to check with you if we still need it :zipper_mouth_face:

Comment (bilelmoussaoui) [reply org.gnome.Lollypop.json:55]: We definitely do, see dhttps://gitlab.freedesktop.org/gstreamer/gst-python/-/issues/40
what's the issue you're getting?

Comment (Eonfge) [reply org.gnome.Lollypop.json:55]:
```
Found ninja-1.10.0 at /usr/bin/ninja
[2/4] Linking target plugin/libgstpython.so
FAILED: plugin/libgstpython.so 
cc  -o plugin/libgstpython.so 'plugin/5e689e2@@gstpython@sha/gstpythonplugin.c.o' -L/app/lib -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libgstpython.so -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wl,-z,relro,-z,now -Wl,--as-needed /usr/lib/x86_64-linux-gnu/libgstreamer-1.0.so /usr/lib/x86_64-linux-gnu/libgobject-2.0.so /usr/lib/x86_64-linux-gnu/libglib-2.0.so /usr/lib/x86_64-linux-gnu/libgstbase-1.0.so -Wl,--export-dynamic /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so -Wl,--end-group -pthread
```
[gst-python.log](https://github.com/flathub/org.gnome.Lollypop/files/5247602/gst-python.log)

**Comment by Eonfge** [comment]
@bilelmoussaoui I've looked into it a bit more, but your Python knowledge certainly eclipses mine, so I guess that you'll have an easier time fixing this.

File: org.gnome.Lollypop.json (hunk 1 above)
Comment (bilelmoussaoui) [reply org.gnome.Lollypop.json]: The issue is caused by gst-python not working properly with python3.8, you will need to backport this https://gitlab.freedesktop.org/gstreamer/gst-python/-/merge_requests/29/diffs for now iirc until the next release of fdo with the new gstreamer release.

Comment (Eonfge) [reply org.gnome.Lollypop.json]: backported the change. Let's hope it works.

**Comment by Eonfge** [comment]
Better late then never. @bilelmoussaoui Good to release?

**Comment by bilelmoussaoui** [comment]
Lgtm, thanks!

**Pull request merged**